test/unit/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -71,65 +70,12 @@ describe('CaseTypeTab (nonprod)', () => {
   });
 });
 
-/**
- * This suite (and sub suites) checks to make sure that there are no duplicate TabDisplayOrder or
- * TabFieldDisplayOrder in the concatenated release definition.
- * Note: As a post release activity the related release describe block can be removed
- * as the feature file would no longer exist
- */
-describe('CaseTypeTab duplicate checks (nonprod)', () => {
```

Comment:
Why are you removing this?

Reply:
I don't see any sense for it. 

`Prod` and `non-prod`. There is no point of testing every single "release" separately.